src/diffpy/cmi/app.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import sys
+
+from diffpy.cmi.version import __version__
+
+
```

Comment:
the reason to use argparse is that it does most of this work for you.  It will print usage and help so you don't need those functions.  Something like
```
    parser = argparse.ArgumentParser(
        description="help/usage message"
    )
    parser.add_argument(
        "--version",
        action="store_true"
        help="Show the program's version number and exit",
    )
   args = parser.parse_args()
   if args.version:
      print("version message")
   else:
       print("help/intro message")
```
should do everything you want